help/sources/tutorials/ui/create/data-partners/acxiom-prospecting-data-import.md

```diff
@@ -0,0 +1,138 @@
+* **Dataflow Details**
+  * **Dataflow Name** - The name this dataflow will appear as in the logging.  By default this will use the name of the file that is being imported
+  * **Description** - Optional description for logging.
+* **Alerts** - Adobe Experience Platform can produce event-based alerts which users can subscribe to, these options all a running dataflow to trigger these.  [Alert overview](../../alerts.md)
+  * **Sources Dataflow Run Start** - Issues an alert when the dataflow starts.
+  * **Sources Dataflow Run Success** - Issues an alert when the dataflow ends without error.
+  * **Sources Dataflow Run Failure** - Issues an alert when the dataflow ends with a failure status.
+<br>    ![Dataset details](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-dataset-details.png)
+
+## Mapping
+
+Defines interface of source columns to associated schema columns on Adobe.  Platform automatically provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases or fix any duplicated mapping fields to clear any errors. [Field mapping in the UI](../../../../../data-prep/ui/mapping.md)
+
+<br>![Mapping](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-mapping.png)
+
+## Schedule your dataflow ingestion
+Defines when the dataflow should be run.
+
+* **Frequency** - Indicates how often the flow should be run, once a week, day, hour, minute.  Or simply to only run once.
+* **Interval** - When a Frequency is selected other than once, interval to establish a set time frame between every ingestion. For example, an ingestion frequency set to Day and an interval set to 15 means that your dataflow is scheduled to ingest data every 15 days.
+* **Start Time** - The timestamp for the projected run, presented in UTC time zone.
+* **Backfill** - A boolean value that determines what data is initially ingested. If backfill is enabled, all current files in the specified path will be ingested during the first scheduled ingestion. If backfill is disabled, only the files that are loaded in between the first run of ingestion and the start time will be ingested. Files loaded prior to start time will not be ingested.
+<br>![Schedualing](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-scheduling.png)
+
+## Review your dataflow
+This presents a file overview of the options selected before submission, allowing users to review the new dataflow before it is created.  Details are grouped within the following categories:
```

Comment:
"Use the review page for a summary of your dataflow prior to ingestion. Details are grouped in the following categories:"

Reply:
Changed descriptive text for the review section to what was provided here.